app/controllers/verify/usps_controller.rb

```diff
@@ -0,0 +1,22 @@
+module Verify
+  class UspsController < ApplicationController
+    include IdvStepConcern
+
+    before_action :confirm_step_needed
+
+    def index
+      @applicant = idv_session.resolution.vendor_resp.normalized_applicant
+    end
+
+    def create
+      idv_session.address_verification_mechanism = :usps
+      redirect_to verify_review_url
```

Comment:
I remembered the reason. The HTTP spec says a redirect should be to an absolute URL, not a relative path. So I just developed that habit a long time ago, before I started using Rails.

This old article seems to confirm my habit https://www.viget.com/articles/rails-named-routes-path-vs-url